app/src/main/res/layout/activity_movie_detail.xml

```diff
@@ -12,84 +12,195 @@
+        <RelativeLayout
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            app:ImageUrl="@{viewModel.backdropPath}"
-            tools:ignore="ContentDescription"
-            />
+            >
 
-        <Button
-            android:id="@+id/button_favorite"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:layout_alignBottom="@+id/image_backdrop"
-            android:layout_alignParentEnd="true"
-            android:layout_alignParentRight="true"
-            android:onClick="@{viewModel::onButtonFavoriteClicked}"
-            android:text="@{viewModel.favoriteText}"
-            tools:ignore="RelativeOverlap"
-            />
+            <FrameLayout
+                android:id="@+id/frame_youtube"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                app:fragmentManager="@{viewModel.fragmentManager}"
+                app:videoKey="@{viewModel.videoKey}"
+                />
 
-        <TextView
-            android:id="@+id/text_tittle"
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:layout_alignParentLeft="true"
-            android:layout_alignParentStart="true"
-            android:layout_below="@+id/image_backdrop"
-            android:gravity="center_horizontal"
-            android:text="@{viewModel.tittle}"
-            android:textColor="@color/color_tuatara"
-            android:textSize="@dimen/sp_16"
-            />
+            <TextView
+                android:id="@+id/tittle"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_alignParentLeft="true"
+                android:layout_alignParentStart="true"
+                android:layout_below="@+id/frame_youtube"
+                android:layout_marginTop="@dimen/dp_14"
+                android:text="@{viewModel.tittle}"
+                android:textColor="@color/color_tuatara"
+                android:textSize="@dimen/sp_16"
+                />
 
-        <TextView
-            android:id="@+id/text_genres"
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:layout_alignParentLeft="true"
-            android:layout_alignParentStart="true"
-            android:layout_below="@+id/text_tittle"
-            android:text="@{viewModel.genres}"
-            />
+            <ImageView
+                android:id="@+id/image_favorite"
+                android:layout_width="@dimen/dp_50"
+                android:layout_height="@dimen/dp_50"
+                android:layout_alignParentEnd="true"
+                android:layout_alignParentRight="true"
+                android:layout_below="@+id/frame_youtube"
+                android:layout_marginRight="@dimen/dp_5"
```

Comment:
WARNING: [RtlHardcoded] Consider adding `android:layout_marginEnd="@dimen/dp_5"` to better support right-to-left layouts
 Using `Gravity#LEFT` and `Gravity#RIGHT` can lead to problems when a layout is rendered in locales where text flows from right to left. Use `Gravity#START` and `Gravity#END` instead. Similarly, in XML `gravity` and `layout_gravity` attributes, use `start` rather than `left`.

For XML attributes such as paddingLeft and `layout_marginLeft`, use `paddingStart` and `layout_marginStart`. *NOTE*: If your `minSdkVersion` is less than 17, you should add *both* the older left/right attributes *as well as* the new start/right attributes. On older platforms, where RTL is not supported and the start/right attributes are unknown and therefore ignored, you need the older left/right attributes. There is a separate lint check which catches that type of error.

(Note: For `Gravity#LEFT` and `Gravity#START`, you can use these constants even when targeting older platforms, because the `start` bitmask is a superset of the `left` bitmask. Therefore, you can use `gravity="start"` rather than `gravity="left|start"`.)